frame/system/src/weights.rs

```diff
@@ -74,3 +65,399 @@ impl ExtrinsicsWeight {
 		*self.get_mut(class) = new;
 	}
 }
+
+/// Block length limit configuration.
+#[derive(RuntimeDebug, Clone)]
+pub struct BlockLength {
+	/// Maximal total length in bytes for each extrinsic class.
+	///
+	/// In the worst case, the total block length is going to be:
+	/// `MAX(max)`
+	pub max: ForDispatchClass<u32>,
```

Comment:
```suggestion
	pub max: ForDispatchClass<Weight>,
```

or just `ExtrinsicsWeight `, which again brings me to noting that I find the type alias to be confusing.